src/lib/cli.ts

```diff
@@ -61,6 +69,23 @@ const setUpUserFeedback = (sonarInstance: Sonar, spinner: IORA) => {
     });
 };
 
+type Update = { // eslint-disable-line no-unused-vars
```

Comment:
It's a bit weird to have this type here. Isn't there a type package for `update-notifier`?
If not I think the type should be declared elsewhere. @sarvaje thoughts?